app/components/check_answers_summary_list_card_component.rb

```diff
@@ -13,7 +13,7 @@ def applicable_questions
   end
 
   def get_answer_label(question)
-    question.answer_label(log).presence || "<span class=\"app-!-colour-muted\">You didn’t answer this question</span>".html_safe
+    question.answer_label(log, user).presence || "<span class=\"app-!-colour-muted\">You didn’t answer this question</span>".html_safe
```

Comment:
Is this meant to be @user?

Reply:
I won't change this then unless it's better practice to do so - @kosiakkatrina @asmega any thoughts on that?

Also presumably if we use `@user` instead of `user` we should also use `@log` instead of `log`?